scripts/deploy/utils/deploy/index.ts

```diff
@@ -0,0 +1,43 @@
+import { execSync } from "child_process";
+import { green } from "../../../utils/colors";
+import { getCurrentCommit, getCurrentlyDeployedCommit } from "../git";
+import { isDryRun } from "../environment";
+
+/**
+ * `runDeploy` is a helper function that actually performs the deploy.
+ * It builds the production bundle, deploys it to the production server, and sends an email.
+ */
+const runDeploy = () => {
+  console.log(green("Getting currently deployed commit"));
```

Comment:
[nit] Green seems unnecessary here, the message isn't indicating any kind of success.